src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -1853,6 +1853,7 @@ internal protected virtual CSharpSyntaxNode GetBindableSyntaxNode(CSharpSyntaxNo
                 case SyntaxKind.OperatorDeclaration:
                 case SyntaxKind.ConversionOperatorDeclaration:
                 case SyntaxKind.GlobalStatement:
+                case SyntaxKind.Subpattern:
```

Comment:
>Subpattern [](start = 32, length = 10)

I didn't understand this either. I'd have expected we needed the containing expression to bind the sub-pattern.
For instance, in `e is { X = 3 }` we need to bind the whole thing including `e`. Or in recursive patterns.

Reply:
The name of this method is perhaps poorly chosen. It returns the syntax that we'd find on a bound node of interest in the map. To get the node in the map we would indeed have to bind the whole statement.

---
In reply to: [191021984](https://github.com/dotnet/roslyn/pull/27160#discussion_r191021984) [](ancestors = 191021984)